addon/pods/components/frost-list/component.js

```diff
@@ -70,6 +69,14 @@ export default Ember.Component.extend({
     })
   }),
 
+  //  Optional attrs for sm vertical-collection
+  //  https://github.com/runspired/smoke-and-mirrors/blob/develop/addon/components/vertical-collection.js
+  idForFirstItem: null,
+  scrollPosition: 0,
+  key: '@identity',
```

Comment:
Ok, so the '@' is more of a placeholder than a suggestion for how to name the property value?  Or is it a necessary format for smoke-and-mirrors?